end-to-end-tests/cypress/fixtures/transfersLinks.json

```diff
@@ -1,42 +1,31 @@
 [
-    "home",
-    "cookie-preferences",
-    "transfers/newtransfersinformation",
-    "transfers/trustname",
-    "transfers/trustsearch?query=company&change=False",
-    "transfers/outgoingtrustdetails?query=SOUTH+YORK+MULTI+ACADEMY+TRUST&change=False&trustId=10067112",
-    "transfers/incomingtrust",
-    "transfers/searchincomingtrust?query=bishop+fraser&change=False",
-    "project/10003000",
-    "project-assignment/10003000",
-    "project/10003000/features",
-    "project/10003000/features/reason",
-    "project/10003000/features/type",
-    "project/10003000/transfer-dates",
-    "project/10003000/transfer-dates/advisory-board-date",
-    "project/10003000/transfer-dates/target-date",
-    "project/10003000/benefits",
-    "project/10003000/benefits/intended-benefits",
-    "project/10003000/benefits/risks",
-    "project/10003000/benefits/equalities-impact-assessment",
-    "project/10003000/legalrequirements",
-    "project/10003000/legalrequirements/outgoing-trust-consent",
-    "project/10003000/legalrequirements/incoming-trust-agreement",
-    "project/10003000/legalrequirements/diocesan-consent",
-    "project/10003000/rationale",
-    "project/10003000/rationale/project",
-    "project/10003000/rationale/trust-or-sponsor",
-    "project/10003000/academy-and-trust-information",
-    "project/10003000/academy-and-trust-information/recommendation",
-    "project/10003000/school-data/10087745",
-    "project/10003000/general-information/10087745",
-    "project/10003000/pupil-number/10087745",
-    "project/10003000/latest-ofsted-judgement/10087745",
-    "project/10003000/latest-ofsted-judgement/10087745?addOrEditAdditionalInformation=true#additional-information-hint",
-    "project/10003000/key-stage-4-performance/10087745",
-    "project/10003000/key-stage-4-performance/10087745?addOrEditAdditionalInformation=true#additional-information-hint",
-    "project/10003000/key-stage-5-performance/10087745",
-    "project/10003000/key-stage-5-performance/10087745?addOrEditAdditionalInformation=true#additional-information-hint",
-    "project/10003000/advisory-board/preview?",
-    "project/10003000/advisory-board/download?" 
-  ]
\ No newline at end of file
+  "home",
+  "cookie-preferences",
+  "transfers/newtransfersinformation",
+  "transfers/trustname",
+  "transfers/trustsearch?query=company&change=False",
+  "transfers/outgoingtrustdetails?query=SOUTH+YORK+MULTI+ACADEMY+TRUST&change=False&trustId=10067112",
+  "transfers/incomingtrust",
+  "transfers/searchincomingtrust?query=bishop+fraser&change=False",
+  "project/10003484",
```

Comment:
Does 10003484 exist on both envs? Or are we at risk of this failing

Reply:
changed the number to one that exist